### PR TITLE
Organize api and Fix OpenAPI spec creation

### LINKED ISSUE #584: Investigate and rectify traceback during OpenAPI spec creation
Post persistent stream changes, our OpenAPI spec creation script is less than pleased. It completes, but less than successfully:

```
$ ./scripts/create_spec 
inactive
Exception ignored in: <function PersistentStream.__del__ at 0x7fa4e7931260>
Traceback (most recent call last):
  File "/home/l4zmq2r/work/repos/AmpliPi/amplipi/streams.py", line 196, in __del__
  File "/home/l4zmq2r/work/repos/AmpliPi/amplipi/streams.py", line 236, in deactivate
Exception: Failed to deactivate Jasons House: 'NoneType' object has no attribute 'kill'
Exception ignored in: <function PersistentStream.__del__ at 0x7fa4e7931260>
Traceback (most recent call last):
  File "/home/l4zmq2r/work/repos/AmpliPi/amplipi/streams.py", line 196, in __del__
  File "/home/l4zmq2r/work/repos/AmpliPi/amplipi/streams.py", line 236, in deactivate
Exception: Failed to deactivate Downstairs: 'NoneType' object has no attribute 'kill'
Generated open API spec to docs/amplipi_api.yaml
```

Investigate & fix.

### PR DESCRIPTION
**Pull request by linknum23**
### What does this change intend to accomplish?
- Organize endpoint documentation in the API explorer on amplipi.local/doc
- Fix #584 (updated script and example file used)
    - NOTE: I was not able to reproduce the exact issue -> @rtertiaer is this still an issue for you?
 
### Checklist
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Does your submission pass linting & tests? You can test on localhost using `./scripts/test`

### CODE REVIEW

File: amplipi/app.py [hunk 1]
```diff
@@ -220,7 +220,8 @@ def shutdown():
 
 
 @api.post(
-  '/api/lms_mode', response_class=Response,
+  '/api/lms_mode', tags=['config'],
```
Comment (rtertiaer) [inline amplipi/app.py:223]: cheers m8

**Review by rtertiaer — APPROVED** [review]
yeah, I no longer get a traceback :sunglasses: 

:shipit:

**Comment by linknum23** [comment]
Does the new endpoint organization make sense to you?

**Comment by rtertiaer** [comment]
It does.

**Pull request merged**